dimes/dimes.py

```diff
@@ -20,6 +20,10 @@ def __init__(
         dimension: Union[str, None] = None,
         color: Union[str, None] = None,
         line_type: Union[str, None] = None,
+        marker_symbol: Union[str, None] = None,
+        marker_size: Union[int, None] = None,
+        marker_line_color: Union[str, None] = None,
+        marker_fill_color: Union[str, None] = None,
```

Comment:
This is getting a little cumbersome. I think we should move these inputs into a new class (maybe a `dataclass`?).

Reply:
@nealkruis I moved these inputs into a dataclass called LineProperties.